surrealdb/http.py

```diff
@@ -333,4 +337,3 @@ async def delete(self, thing: str) -> None:
             method="DELETE",
             uri=f"/key/{table}/{record_id}" if record_id else f"/key/{table}",
         )
-        return response
```

Comment:
Why remove this?

Reply:
The return type-hint is None. Happy to add it back and change the type-hint instead 👍🏼

Reply:
ah, yeah would be good to change the type-hint 👍